From f6cae2a6ccdbd6c8bc19e7d038d7fb4231f063d9 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 25 Jul 2013 18:08:18 +0800
Subject: [PATCH 09/34] ENGR00272459-1 Make sure kernel will be rebuilt for
 different board config

Define KERNEL_CONFIGURE otherwise a clean build on kernel will fail

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 core/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index 84973f8..cfa756c 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -986,6 +986,8 @@ KERNEL_OUT := $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ
 .PHONY: $(TARGET_PREBUILT_KERNEL)
 kernelimage: $(TARGET_PREBUILT_KERNEL)
 
+$(KERNEL_CONFIGURE): kernel_imx/arch/arm/configs/$(TARGET_KERNEL_DEFCONF)
+
 $(TARGET_KERNEL_CONFIGURE): kernel_imx/arch/arm/configs/$(TARGET_KERNEL_DEFCONF) $(KERNEL_CONFIGURE)
 	$(MAKE) -C kernel_imx $(TARGET_KERNEL_DEFCONF) $(KERNEL_ENV)
 	install -D $(KERNEL_CONFIGURE) $(TARGET_KERNEL_CONFIGURE)
-- 
1.8.0

